dockerfiles/e2e-saas/docker-entrypoint.sh

```diff
@@ -139,10 +143,12 @@ export DISPLAY=:1.0
 
 echo "Applying patches..."
 # currently in /tmp/rh-che folder
-patch -u ../e2e/pageobjects/dashboard/Dashboard.ts -i ../test_patches/dashboard.patch 
+patch -u ../e2e/pageobjects/dashboard/Dashboard.ts -i ../test_patches/Dashboard.patch
+patch -u ../e2e/utils/requestHandlers/CheApiRequestHandler.ts -i ../test_patches/CheApiRequestHandler.patch
+patch -u ../e2e/package.json -i ../test_patches/upstream_package.patch
```

Comment:
we should get rid of all the downstream patches. Could you please create an issue for that?

Reply:
I've created an issue https://github.com/eclipse/che/issues/16456 but I'm not sure if we will be able to handle that, as the release should start after 2 working days. I'll do my best.

Reply:
@Katka92 no rush, just please prioritize it for 7.12